src/components/02_atoms/MultipleFields/MultipleFields.js

```diff
@@ -0,0 +1,206 @@
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+import styled from 'react-emotion';
+import List from '@material-ui/core/List';
+import AddIcon from '@material-ui/icons/Add';
+import Button from '@material-ui/core/Button';
+import ListItem from '@material-ui/core/ListItem';
+import DeleteIcon from '@material-ui/icons/Delete';
+import FormLabel from '@material-ui/core/FormLabel';
+import ReorderIcon from '@material-ui/icons/Reorder';
+import FormControl from '@material-ui/core/FormControl';
+import ListItemText from '@material-ui/core/ListItemText';
+import ListItemIcon from '@material-ui/core/ListItemIcon';
+import ListItemSecondaryAction from '@material-ui/core/ListItemSecondaryAction';
+
+const Add = styled('div')`
+  .icon {
+    margin-left: 10px;
+  }
+`;
+
+const style = {
+  listItemStyles: {
+    paddingLeft: 0,
+  },
+  ListItemIconStyles: {
+    cursor: 'move',
+  },
+};
+
+class MultipleFields extends Component {
+  static propTypes = {
+    label: PropTypes.string.isRequired,
+    onChange: PropTypes.func.isRequired,
+    component: PropTypes.func.isRequired,
+    value: PropTypes.arrayOf(PropTypes.string).isRequired,
+  };
+
+  /**
+   * Initial state
+   */
+  state = {
+    currentIndex: -1,
+    selectedValue: null,
+  };
+
+  componentDidMount() {
```

Comment:
Note: I think we need a componentDidUpdate for the [restore content functionality](https://github.com/jsdrupal/drupal-admin-ui/issues/244).

Reply:
I just tried it, and it indeed doesn't work. Do you mind having a look?

Reply:
Does the data actually restore? I get the notification, however, when I click okay nothing happens to any of the fields.